test/spec/controllers/inventoryCtrlSpec.js

```diff
@@ -224,14 +224,56 @@ describe('Inventory Controller', function() {
   });
 
   describe('#hasAllTimeTravelerItems', function() {
-    it('returns false if there are items left in the time traveler store', function() {
+    it('returns false if items remain for purchase with Mystic Hourglasses', function() {
       expect(scope.hasAllTimeTravelerItems()).to.eql(false);
     });
 
     it('returns true if there are no items left to purchase', inject(function(Content) {
       sandbox.stub(Content, 'timeTravelerStore').returns({});
```

Comment:
Fixed. Though my means of figuring out which items to give the `user` are a little kludgey--suggestions welcome!